rosa_getting_started/rosa-config-identity-providers.adoc

```diff
@@ -7,18 +7,16 @@ toc::[]
 
 After your {product-title} (ROSA) cluster is created, you must configure identity providers to determine how users log in to access the cluster.
 
+The following topics describe how to configure an identity provider using the OpenShift Cluster Manager (OCM) console. Alternatively, you can use the `rosa` CLI to create an identity provider and access the cluster.
+
 include::modules/understanding-idp.adoc[leveloffset=+1]
 include::modules/config-github-idp.adoc[leveloffset=+1]
 include::modules/config-gitlab-idp.adoc[leveloffset=+1]
 include::modules/config-google-idp.adoc[leveloffset=+1]
 include::modules/config-ldap-idp.adoc[leveloffset=+1]
 include::modules/config-openid-idp.adoc[leveloffset=+1]
 
-[id="next-steps_rosa-config-identity-providers"]
-== Next steps
-
-xref:../rosa_getting_started/rosa-accessing-cluster.adoc#rosa-accessing-cluster[Access a cluster].
 
 == Additional resources
```

Comment:
In the CLI workflow, you link to this topic showing how to do the same with the console. Does it make sense to likewise link from these console steps to the CLI steps?

Reply:
Hi, Jeana. We do that link to the topic that contains the CLI instructions - the topic is called "Accessing a Cluster" and it covers more than the CLI. Once we can actually link between modules, I can make that clearer with a direct link at the beginning of the topic in context.

Thanks for catching that unique ID piece. We are still retrofitting to make that happen.